lib/main_common.pm

```diff
@@ -2124,7 +2124,12 @@ sub load_security_tests {
 
 sub load_systemd_patches_tests {
     boot_hdd_image;
-    loadtest 'console/systemd_testsuite';
+    if (check_var('SYSTEMD_TESTSUITE', 'basic')) {
+        loadtest 'systemd_testsuite/test_01_basic';
```

Comment:
Are you going to have list of modules here? Alternative could be to pass perl module name in the variable and schedule it. So you won't need to change the code every time you want to change the schedule.

Reply:
I cannot assure now.
At the beginning I was thinking on creating a test suite per test, this is why the created has a specific name, it wouldn't be a list then, but a chain of "if else if".

I am trying to find a better solution by grouping all those test modules in one test suite, but for that I need to provide a clean environment for each one. "openQA snapshots" are not supported for s390x at the moment. (See [poo#46349](https://progress.opensuse.org/issues/46349))

If we can got it, my idea is to actually reuse the current SYSTEMD_TESTSUITE=1, but, until we finish the migration, I don't want to break it. So, at the moment this template is a mix of both. But, at least there is something that @tsaupe can clone and work on.